salt/output/highstate.py

```diff
@@ -275,28 +285,17 @@ def _format_host(host, data):
                     continue
                 if str(ret['result']) in exclude:
                     continue
-            elif __opts__.get('state_output', 'full').lower() == 'terse':
-                # Print this chunk in a terse way and continue in the
-                # loop
+
+            elif any((
```

Comment:
I'm guessing this is a part of the cleanup you mentioned? If so, please break it out into a separate PR. When we mix refactoring and feature work in a single PR it becomes very difficult to manage.